### ISSUE TITLE
Missing name of first feed after executing "Reload all"

### ISSUE THREAD

**Issue by giulianopz** [newsboat/newsboat#2332]
**Newsboat version (copy the output of `newsboat -v` or the first line of `git show`)**:
```
newsboat 2.18.0 - https://newsboat.org/
Copyright (C) 2006-2015 Andreas Krennmair
Copyright (C) 2015-2019 Alexander Batischev
Copyright (C) 2006-2017 Newsbeuter contributors
Copyright (C) 2017-2019 Newsboat contributors

Newsboat is free software licensed under the MIT License. (Type `newsboat -vv' to see the full text.)
It bundles JSON for Modern C++ library, licensed under the MIT License: https://github.com/nlohmann/json

newsboat 2.18.0
System: Linux 5.15.0-58-generic (x86_64)
Compiler: g++ 9.3.0
ncurses: ncurses 6.2.20200212 (compiled with 6.2)
libcurl: libcurl/7.68.0 GnuTLS/3.6.13 zlib/1.2.11 brotli/1.0.7 libidn2/2.2.0 libpsl/0.21.0 (+libidn2/2.2.0) libssh/0.9.3/openssl/zlib nghttp2/1.40.0 librtmp/2.3 (compiled with 7.68.0)
SQLite: 3.31.1 (compiled with 3.31.1)
libxml2: compiled with 2.9.10
```

Config file (copy from ~/.newsboat/config or ~/.config/newsboat/config):
I have no special configurations set, just a list of links
```
:~$ cat .newsboat/urls 
http://antirez.com/rss
https://blog.pragmaticengineer.com/rss/
https://vickiboykis.com/index.xml
https://jvns.ca/atom.xml
https://simonwillison.net/atom/everything/
https://iximiuz.com/feed.rss
https://feeds.feedburner.com/buckblog
https://codeblog.jonskeet.uk/feed/
https://www.schneier.com/feed/atom
https://blog.codinghorror.com/rss/
http://feeds.hanselman.com/ScottHanselman
https://eli.thegreenplace.net/feeds/all.atom.xml
http://dave.cheney.net/feed
https://jeremykun.com/feed/
http://www.norvig.com/rss-feed.xml
https://www.joelonsoftware.com/feed/ 
https://news.ycombinator.com/rss
https://lwn.net/headlines/newrss
https://research.swtch.com/feed.atom
https://martinfowler.com/feed.atom
https://netflixtechblog.com/feed
http://blog.cleancoder.com/atom
http://blog.golang.org/feed.atom
https://beej.us/blog/rss.xml
https://solar.lowtechmagazine.com/feeds/all-en.atom.xml
https://www.hillelwayne.com/index.xml
https://journal.stuffwithstuff.com/rss.xml
https://fasterthanli.me/index.xml
https://matt.might.net/articles/feed.rss
https://joeyh.name/blog/index.rss
https://www.jwz.org/blog/rss
https://bits.debian.org/feeds/feed.rss
https://daniel.haxx.se/blog/feed/
https://randomascii.wordpress.com/rss
https://karpathy.github.io/feed.xml
```

Steps to reproduce the issue:

1. put the above-mentioned urls in the .newsboat/urls

2. launch newboat and type "R" to reload all feeds

3. once reloading is done, the first feed (antirez, in the above list) has no name:
```
newsboat 2.18.0 - Your feeds (35 unread, 35 total)
   1 N   (100/100)
   2 N     (15/15) The Pragmatic Engineer
   3 N     (20/20) Tech Blog on ★❤✰ Vicki Boykis ★❤✰
   4 N     (20/20) Julia Evans
   5 N     (30/30) Simon Willison's Weblog
   6 N     (83/83) Ivan on Containers, Kubernetes, and Server-Side
   7 N     (10/10) The Buckblog
   8 N     (10/10) Jon Skeet's coding blog
   9 N     (10/10) Schneier on Security
  10 N     (15/15) Coding Horror
  11 N     (11/11) Scott Hanselman's Blog
  12 N     (10/10) Eli Bendersky's website
  13 N     (20/20) Dave Cheney
  14 N       (5/5) Math ∩ Programming
  15 N     (77/77) Peter Norvig
  16 N     (10/10) Joel on Software
  17 N     (30/30) Hacker News
  18 N     (15/15) LWN.net
  19 N     (21/21) research!rsc
  20 N     (30/30) Martin Fowler
  21 N     (10/10) Netflix TechBlog - Medium
  22 N     (20/20) The Clean Code Blog
  23 N     (10/10) The Go Blog
  24 N     (37/37) Beej's Bit Bucket
  25 N       (3/3) LOW←TECH MAGAZINE
  26 N     (20/20) Hillel Wayne
  27 N     (10/10) journal.stuffwithstuff.com
  28 N     (25/25) fasterthanli.me
  29 N     (10/10) Matt Might's blog
  30 N     (10/10) see shy jo
  31 N     (30/30) jwz
  32 N   (242/242) Bits from Debian
  33 N     (10/10) daniel.haxx.se
  34 N     (10/10) Random ASCII – tech blog of Bruce Dawson
  35 N     (10/10) Andrej Karpathy blog
```

**Comment by der-lyse** [comment]
You're running an old version of Newsboat. It seems to be fixed in the meantime, I tested with version 2.30 and the feed title is correctly shown as `<antirez>`.

**Comment by giulianopz** [comment]
> You're running an old version of Newsboat. It seems to be fixed in the meantime, I tested with version 2.30 and the feed title is correctly shown as `<antirez>`.

Thanks, upgrading the version fixed my issue.

**Comment by Minoru** [comment]
For anyone who stumbles upon this in the future, this bug was fixed in Newsboat 2.19: https://github.com/newsboat/newsboat/issues/796.
